### PR TITLE
#Task27 Feat/ Update footer-responsive 

### PR DESCRIPTION
**Pull request by TheHienDuong**
[#Task31](https://trello.com/c/3NPddzDN/19-task31-apply-bootstrap-class-into-cta-section) Apply bootstrap class into CTA-section
- Update background
- update grid responsive]
- Using mobile first to responsive web cta
Using class bootstrap naming .

### Overview

https://user-images.githubusercontent.com/113186169/224256767-bcb484b6-fb2a-4121-aa91-51c9690301fa.mp4

### CODE REVIEW

File: practice-02/index.html [hunk 1]
```diff
@@ -467,94 +467,98 @@ <h1 class="heading-lg" href="#">
...
-        <div class="footer__rule text--black container ">
-            <a class="rule__copyright" href="__blank">&#169;Copyright 2022 Yoora, Inc.</a>
-
-            <div class="footer__rule-right">
-                <ul class="rule__menu menu__list">
-                    <li><a class="item" href="#">Terms of Service</a></li>
-                    <li><a class="item" href="#">Privacy Police</a></li>
-                    <li><a class="item" href="#">Cookies</a></li>
+        <hr class="line ">
+        <div class="footer-about container">
```
Comment (ThanhPhat1080) [inline practice-02/index.html]: find another name

File: practice-02/styles/index.css [hunk 2]
```diff
@@ -826,9 +833,50 @@ a {
...
 
+  /******************/
+  /* footer */
+  .footer-content {
+    grid-template-columns: repeat(6,1fr);
+  }
+  .footer-content .card{
+    grid-column: 1/3;
+  }
+    .footer-about .menu-list {
```
Comment (ThanhPhat1080) [inline practice-02/styles/index.css]: check indent

File: practice-02/styles/index.css [hunk 3]
```diff
@@ -840,6 +888,46 @@ a {
...
+      grid-template-rows: unset;
+      justify-items: start;
+      align-items: end;
+      padding-bottom: 0;
+    }
+    .footer-locale {
+      grid-area: 1/3;
+    }
+    .footer-about .nav-menu .item {
+      /* margin-bottom: ; */
```
Comment (ThanhPhat1080) [inline practice-02/styles/index.css]: remove

Comment (ThanhPhat1080) [reply practice-02/styles/index.css]: remove unusual classname too

**Review by phattruong-agilityio — APPROVED** [review]

**Review by phattruong-agilityio — APPROVED** [review]

**Pull request merged**